tests/source/issue-1124.rs

```diff
@@ -0,0 +1,3 @@
+use d; use c; use b; use a; 
```

Comment:
I'd like to see some tests with blanks lines between imports and trailing whitespace before a newline